cmd/wallet/makeTransaction/makeTransaction.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/configwizard/greenfinch-sdk/pkg/wallet"
+	client "github.com/nspcc-dev/neo-go/pkg/rpcclient"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"io/ioutil"
+	"log"
+	"math/big"
+	"os"
+	"time"
+)
+
+const NEOFS_CONTRACT = "0x0a81b80376a65003781f140d1b87b6531f706215" //todo is this still the correct contract
+const usage = `Example
+
+$ ./balance -wallets ./sample_wallets/wallet.rawContent.go
+password is password
+`
+
+var (
+	walletPath = flag.String("wallet", "", "path to JSON wallets file")
+	walletAddr = flag.String("address", "", "wallets address [optional]")
+	createWallet = flag.Bool("create", false, "create a wallets")
+	password = flag.String("password", "", "wallet password")
+)
+
+func main() {
+	path, err := os.Getwd()
+	if err != nil {
+		log.Println(err)
+	}
+	fmt.Println(path)  // for example /home/user
+	flag.Usage = func() {
+		_, _ = fmt.Fprintf(os.Stderr, usage)
+		flag.PrintDefaults()
+	}
+	flag.Parse()
+
+	ctx := context.Background()
+
+	if *createWallet {
+		secureWallet, err := wallet.GenerateNewSecureWallet(*walletPath, "some account label", *password)
+		if err != nil {
+			log.Fatal("error generating wallets", err)
+		}
+		file, _ := json.MarshalIndent(secureWallet, "", " ")
+		_ = ioutil.WriteFile(*walletPath, file, 0644)
+		log.Printf("created new wallets\r\n%+v\r\n", file)
+		os.Exit(0)
+	}
+
+	cli, err := client.New(ctx, string(wallet.RPC_TESTNET), client.Options{})
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = cli.Init()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	acc, err := wallet.UnlockWallet(*walletPath, "", *password)
+	if err != nil {
+		log.Fatal("can't unlock wallet:", err)
+	}
+
+	account, err := wallet.StringToUint160(acc.Address)
+	param := smartcontract.Parameter{
+		Type:  smartcontract.Hash160Type,
+		Value: account,
+	}
+	params := []smartcontract.Parameter{param}
+	operation := "balanceOf" //symbol
+	network := wallet.RPC_TESTNET
+	transactionID, transaction, err := wallet.CreateTransactionFromFunctionCall(NEOFS_CONTRACT, operation, network, acc, params)
```

Comment:
A transaction for `balanceOf` call?

Reply:
This example is over a year old (the principle of it) - I can' t remember why I chose that but its meant to be an example as i don't have a contract on a network that I can run a real transaction against